baseapp/abci_utils.go

```diff
@@ -38,8 +39,8 @@ type (
 	// extension signatures. Typically, this will be implemented by the x/staking
 	// module, which has knowledge of the CometBFT public key.
 	ValidatorStore interface {
-		GetValidatorByConsAddr(sdk.Context, cryptotypes.Address) (Validator, error)
-		TotalBondedTokens(ctx sdk.Context) math.Int
+		GetValidatorByConsAddr(context.Context, cryptotypes.Address) (Validator, error)
```

Comment:
Any reason this isn't `sdk.ConsAddress`?

Reply:
Maybe @alexanderbez knows, because I don't 😅